hub/tests/path_fixtures.py

```diff
@@ -155,3 +155,30 @@ def flower_path():
 
     path = get_dummy_data_path("compressed_images")
     return os.path.join(path, "flower.png")
+
+
+@pytest.fixture
+def compressed_image_paths():
+    paths = {
+        "webp": "beach.webp",
+        "gif": "boat.gif",
+        "bmp": "car.bmp",
+        "jpeg": "cat.jpeg",
+        "wmf": "crown.wmf",
+        "dib": "dog.dib",
+        "tiff": "field.tiff",
+        "png": "flower.png",
+        "ico": "sample_ico.ico",
+        "jpeg2000": "sample_jpeg2000.jp2",
+        "pcx": "sample_pcx.pcx",
+        "ppm": "sample_ppm.ppm",
+        "sgi": "sample_sgi.sgi",
+        "tga": "sample_tga.tga",
+        "xbm": "sample_xbm.xbm",
```

Comment:
all of these images add > 2MB of data to the repo, could you get some smaller images or tiny synthetic images?